src/graph_notebook/magics/graph_magic.py

```diff
@@ -91,6 +91,32 @@
 RDF_LOAD_FORMATS = [FORMAT_NTRIPLE, FORMAT_NQUADS, FORMAT_RDFXML, FORMAT_TURTLE]
 BASE_URI_FORMATS = [FORMAT_RDFXML, FORMAT_TURTLE]
 
+MEDIA_TYPE_SPARQL_JSON = "application/sparql-results+json"
+MEDIA_TYPE_SPARQL_XML = "application/sparql-results+xml"
+MEDIA_TYPE_BINARY_RESULTS_TABLE = "application/x-binary-rdf-results-table"
+MEDIA_TYPE_SPARQL_CSV = "text/csv"
+MEDIA_TYPE_SPARQL_TSV = "text/tab-separated-values"
+MEDIA_TYPE_BOOLEAN = "text/boolean"
+MEDIA_TYPE_NQUADS = "application/n-quads"
+MEDIA_TYPE_NQUADS_TEXT = "text/x-nquads"
+MEDIA_TYPE_RDF_XML = "application/rdf+xml"
+MEDIA_TYPE_JSON_LD = "application/ld+json"
+MEDIA_TYPE_NTRIPLES = "application/n-triples"
+MEDIA_TYPE_NTRIPLES_TEXT = "text/plain"
+MEDIA_TYPE_TURTLE = "text/turtle"
+MEDIA_TYPE_N3 = "text/n3"
+MEDIA_TYPE_TRIX = "application/trix"
+MEDIA_TYPE_TRIG = "application/trig"
+MEDIA_TYPE_RDF4J_BINARY = "application/x-binary-rdf"
+
+NEPTUNE_RDF_SELECT_FORMATS = [MEDIA_TYPE_SPARQL_JSON, MEDIA_TYPE_SPARQL_XML, MEDIA_TYPE_BINARY_RESULTS_TABLE,
+                              MEDIA_TYPE_SPARQL_CSV, MEDIA_TYPE_SPARQL_TSV]
+NEPTUNE_RDF_ASK_FORMATS = [MEDIA_TYPE_SPARQL_JSON, MEDIA_TYPE_SPARQL_XML, MEDIA_TYPE_BOOLEAN]
+NEPTUNE_RDF_CONSTRUCT_DESCRIBE_FORMATS = [MEDIA_TYPE_SPARQL_JSON, MEDIA_TYPE_NQUADS, MEDIA_TYPE_NQUADS_TEXT,
```

Comment:
Are we certain that ALL these result formats are supported ?

Reply:
I believe, for Neptune, those are the ones in the documentation. I'm not sure we have verified that each works, but they are documented as working.

Reply:
I have manually verified that each result format listed returns a valid result when used with the corresponding query type on Neptune, tested on Release 1.1.0.0.